integration_tests/src/main/python/delta_lake_low_shuffle_merge_test.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pyspark.sql.functions as f
+import pytest
+
+from delta_lake_merge_common import *
+from marks import *
+from pyspark.sql.types import *
+from spark_session import *
+
+delta_merge_enabled_conf = copy_and_update(delta_writes_enabled_conf,
+                                           {"spark.rapids.sql.command.MergeIntoCommand": "true",
+                            "spark.rapids.sql.command.MergeIntoCommandEdge": "true",
+                            "spark.rapids.sql.delta.lowShuffleMerge.enabled": "true",
+                            "spark.rapids.sql.format.parquet.reader.type": "PERFILE"})
+
+@allow_non_gpu("RapidsRepartitionByFilePathExec", *delta_meta_allow)
```

Comment:
How does RapidsRepartionByFilePathExec work if it isn't converted to GpuRapidsRepartitionByFilePathExec?  The former has a doExecute method that will throw, so seems like this would just crash in practice if it wasn't converted to the GPU version.

Reply:
> It's not actually executed

Curious how is it not executed in practice?  I thought it would be a child node of the join, as that's the whole point of it to repartition without actually shuffling for the join.  The join is going to try to execute it's child nodes to perform the join, which seems fatal if the CPU RapidsRepartitionByFilePathExec is in the plan.  The GPU version of the node has an execute (columnar) implementation, so that makes it even more confusing how the CPU version does not need one.